.github/workflows/build-domain.yaml

```diff
@@ -126,30 +128,43 @@ jobs:
           overrides="${service}/chart-overrides.yaml"
           name=$(basename $(yq '.name' "$overrides"))
 
-          version=$(yq '.version' "$overrides")
-          version=$(sh ./.github/tooling/semver.sh "$version" "$PR")
+          tag=$(yq '.version' "$overrides")
+          tag=$(sh ./.github/tooling/semver.sh "$tag" "$PR")
 
-          echo "service=${service}" >> $GITHUB_OUTPUT
-          echo "project=${PROJECT}" >> $GITHUB_OUTPUT
-          echo "docker=${docker}" >> $GITHUB_OUTPUT
-          echo "domain=${domain}" >> $GITHUB_OUTPUT
-          echo "version=${version}" >> $GITHUB_OUTPUT
-          echo "name=${name}" >> $GITHUB_OUTPUT
+          dotnet=${{ matrix.build.dotnetVersion }}
+
+          {
+            echo "service=${service}"
+            echo "project=${PROJECT}"
+            echo "docker=${docker}"
+            echo "domain=${domain}"
+            echo "tag=${tag}"
+            echo "name=${name}"
+            echo "dotnet=${dotnet::${#dotnet}-2}"
+          } >> $GITHUB_OUTPUT
+
+      - name: Pin dotnet version
+        shell: bash
+        working-directory: ${{ steps.resolver.outputs.service }}
+        run: echo '{"sdk":{"rollForward":"disable","version":"${{ inputs.dotnet-version }}"}}' > global.json
```

Comment:
This not output the correct version. An empty string is outputted.

Reply:
It should be the value from the matrix, yes, and it is now.